src/NuGet/Microsoft.Net.Compilers.Toolset/Framework/build/Microsoft.Net.Compilers.Toolset.Framework.props

```diff
@@ -0,0 +1,12 @@
+﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+
+  <!-- 
+    This packag is meant to override the compiler when running on .NET Framework MSBuild. When running
```

Comment:
```suggestion
    This package is meant to override the compiler when running on .NET Framework MSBuild. When running
```

Reply:
hahah ... yeah i'm terrible at the spelling thing.